solr/core/src/java/org/apache/solr/core/CoreContainer.java

```diff
@@ -192,11 +193,16 @@ public CoreLoadFailure(CoreDescriptor cd, Exception loadFailure) {
       new PluginBag<>(SolrRequestHandler.class, null);
 
   private volatile ApplicationHandler jerseyAppHandler;
+  private volatile JerseyAppHandlerCache appHandlersByConfigSetId;
 
   public ApplicationHandler getJerseyApplicationHandler() {
     return jerseyAppHandler;
   }
 
+  public JerseyAppHandlerCache getAppHandlerCache() {
```

Comment:
this name `getAppHandlerCache()` is not clear enough. please consider `getJerseyHandlers()` or something like that

Reply:
I added Jersey to the name - that was something David suggested on the previous PR.